java/client/src/main/java/glide/api/models/commands/RangeOptions.java

```diff
@@ -316,7 +316,7 @@ public static String[] createZRangeStoreArgs(
                 new String[] {destination, source}, createZRangeBaseArgs(rangeQuery, reverse));
     }
 
-    private static String[] createZRangeBaseArgs(RangeQuery rangeQuery, boolean reverse) {
+    public static String[] createZRangeBaseArgs(RangeQuery rangeQuery, boolean reverse) {
```

Comment:
why public?
Please add a doc if it is needed to be public.

Reply:
I am using it directly while using the new `newArgsBuilder()` (it simplifies things)
Will add doc